File: Day 8/MultiScreen_1/app/src/main/java/com/example/multiscreen_1/MainActivity.java
package com.example.multiscreen_1;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    EditText name;

    public static final String EXTRA_NAME = "1";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }

    public void openActivity(View v )
    {
        Toast.makeText(this, "Loading", Toast.LENGTH_SHORT).show();

        Intent i = new Intent(this,MainActivity2.class);

        name=findViewById(R.id.name);
        String nameText = name.getText().toString();
        i.putExtra(EXTRA_NAME,nameText);

        startActivity(i);
    }

}

File: Siddhi mam Tasks/Day 37/Co-routines/ViewModelScopeLifeCycleScope/app/src/main/java/com/example/viewmodelscopelifecyclescope/MainActivity.kt
package com.example.viewmodelscopelifecyclescope

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.lifecycle.ViewModelProvider
import androidx.lifecycle.lifecycleScope
import kotlinx.coroutines.delay
import kotlinx.coroutines.launch

class MainActivity : AppCompatActivity() {


    lateinit var viewModel: MainViewModel


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        viewModel = ViewModelProvider(this).get(MainViewModel::class.java)


        lifecycleScope.launch{
            delay(2000)
            val intent = Intent(this@MainActivity,AnotherActivity::class.java)
            startActivity(intent)
            finish()
        }


    }
}